modules/update-using-custom-machine-config-pools-inheritance.adoc

```diff
@@ -0,0 +1,208 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/update-using-custom-machine-config-pools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-using-custom-machine-config-pools-mcp-inheritance_{context}"]
+= Managing machine configuration inheritance for a worker pool canary
+
+You can configure a machine config pool (MCP) canary to inherit any `MachineConfig` assigned to an existing MCP. 
+This configuration is useful when you want to use an MCP canary to test as you update the nodes of an existing MCP, one at a time.
+
+
+.Prerequisites
+
+* You have created one or more custom machine config pools (MCP).
+
+.Procedure
+
+. Create a secondary MCP as described in the following two steps:
+.. Save the following configuration file as `machineConfigPool.yaml`.
++
+.Example `machineConfigPool` YAML
+[source,yaml]
+----
```

Comment:
Which parts of this sample have to stay as-is and which might the user tinker with?

Reply:
Things such as names can be changed - this is just an example - but any change here would have to be replicated in subsequent steps that refer back to the named MCP.